cypress/e2e/dev/5-management-tests/no-autodatastore-on-management-pages.cypress.js

```diff
@@ -55,9 +55,9 @@ describe('clear data page', () => {
 
     cy.task('log', 'Add some data')
     cy.visit(`/question-check?most-impressive-trick=${encodeURIComponent(answer)}`)
-    cy.get('.govuk-header__logotype-text')
+    cy.get('.govuk-header__logotype')
```

Comment:
I've changed the logo to use the parent `.govuk-header__logotype`

This way they'll work with both v4 and v5 should we need to switch